### PR TITLE
Bring the Last time results filter back to the monitor page

### LINKED ISSUE #7819: missing "show last build results" in ui
# Issue Description

Monitor project page in the "new Beta" ui is missing the "Last time results:" checkbox (or some equivalent). which makes it very much less useful. At least I'm not able to find it in the new UI.

The feature itself still seems to be there by adding &lastbuild=1

Expected Result
============

Some way in the new UI to show the last build results like the old UI provided via a checkbox.

Further Information
===============
* A link to an OBS instance showing the issue (tested in IBS)

### PR DESCRIPTION
**Pull request by vpereira**
Fixes #7819 

Screenshot_2019-07-02_10-16-50

### CODE REVIEW

**Comment by saraycp** [comment]
I was about to say that having a checkbox inside a kind of button looks weird to me... but immediately after I saw this in GitHub:
Screenshot-2019-7-2 openSUSE open-build-service
 
Anyway, it's a good solution. If somebody else agree, I agree too.

File: src/api/app/views/webui2/webui/project/_monitor_control.html.haml [hunk 1]
```diff
@@ -44,6 +44,12 @@
                 = check_box_tag valid_xml_id('repo_' + repository), 1, repository_filter.include?(repository), class: 'custom-control-input'
                 %label.custom-control-label{ for: valid_xml_id('repo_' + repository) }
                   = repository
+      %span#project-monitor-repositories-last-build
+        %button.btn.btn-outline-secondary
+          = check_box_tag 'lastbuild', 1, lastbuild_switch
+          Last time results
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/project/_monitor_control.html.haml]: :thought_balloon: 

```
<span title="">Include the results from the last build</span>
```

Comment (vpereira) [reply src/api/app/views/webui2/webui/project/_monitor_control.html.haml]: @hennevogel do you want to switch the "last time results" by "Include the results from the last build" ? I'm wondering if it's not too long?

Comment (saraycp) [reply src/api/app/views/webui2/webui/project/_monitor_control.html.haml]: I think he wants you to add a title, so when you do mouseover you can read the long text.

Comment (saraycp) [reply src/api/app/views/webui2/webui/project/_monitor_control.html.haml]: Oops or not! That's what I understood :thinking:

Comment (DavidKang) [reply src/api/app/views/webui2/webui/project/_monitor_control.html.haml]: > @hennevogel do you want to switch the "last time results" by "Include the results from the last build" ? I'm wondering if it's not too long?

@vpereira, I think he is telling you to add `title='Include the results from the last build'`

**Comment by eduardoj** [comment]
When you click in the button, but not in the checkbox, the checkbox doesn't get checked.

simplescreenrecorder-2019-07-02_12 41 56_monitor_last_time_results

**Comment by vpereira** [comment]
> When you click in the button, but not in the checkbox, the checkbox doesn't get checked.

just if you click in the checkbox.

File: src/api/app/views/webui2/webui/project/_monitor_control.html.haml (hunk 1 above)
Comment (hennevogel) [reply src/api/app/views/webui2/webui/project/_monitor_control.html.haml]: yeah, some long for so you can actually understand what this does. What does it do BTW? ;-)

Comment (vpereira) [reply src/api/app/views/webui2/webui/project/_monitor_control.html.haml]: Screenshot_2019-07-02_14-23-06

**Comment by vpereira** [comment]
> When you click in the button, but not in the checkbox, the checkbox doesn't get checked.

could you please review it again @eduardoj ?

**Comment by eduardoj** [comment]
> > When you click in the button, but not in the checkbox, the checkbox doesn't get checked.
> 
> could you please review it again @eduardoj ?

LGTM. Thanks!

**Pull request merged**